src/handlers/scheduledEventHandler.ts

```diff
@@ -1,8 +1,60 @@
+import { KVNamespace } from '@cloudflare/workers-types';
+
 import { handleConfig } from '../config/config';
-import { env } from '../types/global.types';
+import { NAMESPACE_NAME } from '../constants';
+import { env, nicknameUpdateResponseType } from '../types/global.types';
+import { generateJwt } from '../utils/generateJwt';
 
 export async function ping(env: env) {
 	const url = handleConfig(env);
 	const response = await fetch(`${url.baseUrl}/healthcheck`);
 	return response;
 }
+
+export async function callDiscordNicknameBatchUpdate(env: env) {
+	const namespace = env[NAMESPACE_NAME] as unknown as KVNamespace;
+	let lastNicknameUpdate: string | number | null = 0;
```

Comment:
Earlier I was type casting to number and then assigning it to `lastNicknameUpdate` so added it here. It will always either be a string or null if the key isn't present. I'll make the change.